extension/src/experiments/data/index.ts

```diff
@@ -74,7 +79,7 @@ export class ExperimentsData extends BaseData<ExperimentsOutput> {
       this.dvcRoot
     )
 
-    const dotGitPath = getGitPath(gitRoot, gitPath.DOT_GIT)
+    const dotGitPath = this.dotGitPath || getGitPath(gitRoot, gitPath.DOT_GIT)
```

Comment:
[Q] Are you sure you need this when `AvailableCommands.GIT_GET_REPOSITORY_ROOT` is already returning the mockRoot? Could you stub `getGitPath` instead of adding code just for testing purposes?

Reply:
Good idea!